src/NUnitFramework/tests/Attributes/OneTimeSetUpTearDownTests.cs

```diff
@@ -229,7 +229,7 @@ public void HandleErrorInFixtureTearDownAfterErrorInTest()
             Assert.AreEqual(1, fixture.setUpCount, "setUpCount");
             Assert.AreEqual(1, fixture.tearDownCount, "tearDownCount");
 
-            Assert.AreEqual("One or more child tests had errors" + Env.NewLine + "TearDown : System.Exception : This was thrown from fixture teardown", result.Message);
+            Assert.AreEqual(TestResult.CHILD_ERRORS_MESSAGE + Env.NewLine + "TearDown : System.Exception : This was thrown from fixture teardown", result.Message);
```

Comment:
This is out of scope for your change but it might be good to assert on the value of Result.FailureSite wherever we are asserting on CHILD_ERRORS_MESSAGE. We can get that at a later time though.